dotcom-rendering/src/layouts/FrontLayout.tsx

```diff
@@ -191,6 +213,13 @@ export const FrontLayout = ({ front, NAV }: Props) => {
 					showAge={false}
 					absoluteServerTimes={absoluteServerTimes}
 					imageLoading="eager"
+					aspectRatio={
+						highlightsCollection.aspectRatio
+							? highlightsCollection.aspectRatio
+							: fallBackAspectRatio(
+									highlightsCollection.collectionType,
+							  )
+					}
```

Comment:
You could use nullish coalescing for these rather than a ternary:

```ts
highlightsCollection.aspectRatio ?? fallBackAspectRatio(highlightsCollection.collectionType)
```